src/server/game/Entities/Player/Player.cpp

```diff
@@ -6151,16 +6151,11 @@ void Player::CheckAreaExploreAndOutdoor()
     if (IsInFlight())
         return;
 
-    bool isOutdoor;
-    uint32 areaId = GetMap()->GetAreaId(GetPhaseShift(), GetPositionX(), GetPositionY(), GetPositionZ(), &isOutdoor);
-    AreaTableEntry const* areaEntry = sAreaTableStore.LookupEntry(areaId);
-
-    if (sWorld->getBoolConfig(CONFIG_VMAP_INDOOR_CHECK) && !isOutdoor)
+    if (sWorld->getBoolConfig(CONFIG_VMAP_INDOOR_CHECK) && !IsOutdoors())
         RemoveAurasWithAttribute(SPELL_ATTR0_OUTDOORS_ONLY);
 
-    if (!areaId)
-        return;
-
+    uint32 const areaId = GetMap()->GetAreaId(GetPhaseShift(), GetPositionX(), GetPositionY(), GetPositionZ());
```

Comment:
I don't know if this is correct.

in 335: https://github.com/TrinityCore/TrinityCore/commit/e79c595b695ea5f35b8324efbb7741ccb102bce9#diff-baadebd8cd1117ca48225f316a5ab3fd5fd55b20963394d302341147183db067R6385

Reply:
No, it is not, it should be GetAreaId(), just like in 3.3.5